api/construction_service.go

```diff
@@ -500,28 +529,34 @@ func (s *Server) ConstructionPreprocess(ctx context.Context, r *types.Constructi
 		}
 		opts.SequenceNumber = seq
 	}
-	val, ok := r.Metadata["payer"]
-	if !ok {
-		return nil, wrapErrorf(
-			errInvalidMetadataField,
-			"payer metadata field is missing",
-		)
-	}
-	raw, ok := val.(string)
-	if !ok {
-		return nil, wrapErrorf(
-			errInvalidMetadataField,
-			"payer metadata field is not a string: %v", val,
-		)
-	}
-	payer, xerr := s.getAccount(raw)
-	if xerr != nil {
-		return nil, wrapErrorf(
-			errInvalidMetadataField,
-			"invalid payer value: %s", formatErr(xerr),
-		)
+	// NOTE(tav): We override the metadata.payer with the sender account
```

Comment:
Does it mean that Nova does not need to pass sender addr to payer for proxy_transfer_inner?

Reply:
Right.